docs/reference/edot-sdks/android/configuration.md

```diff
@@ -0,0 +1,380 @@
+3. Your app's build type, flavor, backend environment, or a combination of these. Helpful to better analyze your app's data later in {{kib}}.
+
+### Export connectivity
+
+Configure where your app's telemetry is exported:
+
+```kotlin
+class MyApp : android.app.Application {
+
+    override fun onCreate() {
+        super.onCreate()
+        val agent = ElasticApmAgent.builder(this)
+            // ...
+            .setExportUrl("https://my-elastic-apm-collector.endpoint") // <1>
+            .setExportAuthentication(Authentication.ApiKey("my-api-key")) // <2>
+            .setExportProtocol(ExportProtocol.HTTP) // <3>
+            .build()
+    }
+}
+```
+
+1. Your endpoint URL. If you don't have one yet, refer to [Get the export endpoint](getting-started.md#get-export-endpoint).
+2. Your authentication method. You can use either an [API key](docs-content://solutions/observability/apm/api-keys.md), a [Secret token](docs-content://solutions/observability/apm/secret-token.md), or none. Defaults to `None`. API keys are the recommended method, if you don't have one yet, refer to [Create an API key](getting-started.md#create-api-key).
+3. The protocol used to communicate with your endpoint. It can be either `HTTP` or `gRPC`. Defaults to `HTTP`.
+
+:::{tip}
+If you'd like to provide these values from outside of your code, using an environment variable or a properties file for example, refer to [Provide config values outside of your code](configuration.md#provide-config-values-from-outside-of-your-code).
+:::
+
+### Intercept export request headers
+
+You can provide an interceptor for the signals' export request headers, where you can read or modify them if needed.
+
+```kotlin
+class MyApp : android.app.Application {
+
+    override fun onCreate() {
+        super.onCreate()
+        val agent = ElasticApmAgent.builder(this)
+            // ...
+            .setExportHeadersInterceptor(interceptor)
+            .build()
+    }
+}
+```
+
+### Intercept attributes
+
+You can provide global interceptors for all spans and logs [attributes](https://opentelemetry.io/docs/specs/otel/common/#attribute). Interceptors are executed on every span or log creation, where you can read or modify them if needed.
+
+This is useful for setting dynamic global attributes. If you'd like to set static global attributes, which are also applied to metrics, take a look at [Intercept resources](#intercept-resources).
+
+```kotlin
+class MyApp : android.app.Application {
+
+    override fun onCreate() {
+        super.onCreate()
+        val agent = ElasticApmAgent.builder(this)
+            // ...
+            .addSpanAttributesInterceptor(interceptor)
+            .addLogRecordAttributesInterceptor(interceptor)
+            .build()
+    }
+}
+```
+
+### Session behavior
+
+You can configure how sessions work. For example:
```

Comment:
I noticed that sessions aren't a clear topic for a lot of people, which is why I added a link on the word `sessions` in the existing docs that takes users to the feature definition on the index page. Can we add it here too?